### PR TITLE
Update main install docs

### PR DESCRIPTION
**Pull request by marcoag**
Updating the main installation instructions for Open-RMF to prioritize building from source. Also adding the missing 22.09 release link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,22 +9,25 @@ The Open-RMF platform for multi-fleet robot management.
...
 | RMF Version | Installation Instructions                                                        | Supported distros                                    | Supported ROS2 versions |
 | ----------- | -------------------------------------------------------------------------------- | ---------------------------------------------------- | ----------------------- |
-| 21.09       |  [Installation instructions](https://github.com/open-rmf/rmf/tree/release/21.09) | Ubuntu 20.04, Ubuntu 21.09, RHEL 8 (deployment only) | Foxy, Galactic          |
+| 22.09       |  [Install RMF 22.09](https://github.com/open-rmf/rmf/tree/release/22.09) | Ubuntu 22.04, RHEL 8 (deployment only) | Humble          |
+| ~~21.09~~ (EOL) |  ~~[Install RMF 21.09](https://github.com/open-rmf/rmf/tree/release/21.09)~~ | ~~Ubuntu 20.04, Ubuntu 21.09, RHEL 8 (deployment only)~~ | ~~Foxy, Galactic~~        |
+
+## Installation Instructions
+
+Even though binary versions are avilable, given the fast development and frequent updates to Open-RMF,
+it is hevily recommended to follow instructions here and **install from
```
Comment (Yadunund) [inline README.md:22]:
```suggestion
it is heavily recommended to follow instructions here and **install from
```

Comment (Yadunund) [reply README.md:22]: Do we want to link a section from  below when we say "here"?

**Review by Yadunund — CHANGES_REQUESTED** [review]
Minor nitpick, but lgtm

File: README.md (hunk 1 above)
Comment (marcoag) [reply README.md]: I had a link there to the "source installation" section but then I realized that might make people skip the installation of ROS and Gazebo before trying to install RMF from source so I removed it. Maybe reword the `here` into `succeeding` or `subsequent`?

Comment (Yadunund) [reply README.md]: Maybe "follow instructions below and .."?

Comment (marcoag) [reply README.md]: Yup, sounds good, added.

**Review by Yadunund — APPROVED** [review]
